core/src/main/web/app/helpers/evaluatepluginmanager.js

```diff
@@ -129,9 +129,18 @@
                   .then(function(factory) {
                     if (factory !== undefined && factory.create !== undefined) {
                       return factory.create(evaluatorSettings).then(function(ev) { deferred.resolve(ev); });
-                    } else
-                      deferred.reject("no factory for evaluator plugin");
+                    } else {
+		      // or should someone pass in a scope to use, like with a method on the evaluatorLoadQueue?
+                      $rootScope.pluginId = name;
```

Comment:
i tried a few things like that but did not fly.  would be happy to see it though.